portability-spi-cloud/src/test/java/org/dataportabilityproject/spi/cloud/types/PortabilityJobTest.java

```diff
@@ -0,0 +1,43 @@
+package org.dataportabilityproject.spi.cloud.types;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.time.LocalDateTime;
+import org.dataportabilityproject.spi.cloud.types.PortabilityJob.State;
+import org.dataportabilityproject.test.types.ObjectMapperFactory;
+import org.junit.Test;
+
+/**
+ * Tests serialization and deserialization of a {@link PortabilityJob}.
+ */
+public class PortabilityJobTest {
```

Comment:
note: the rename of this class is because there used to be an extra "main" in the path (src/test/main/java/...)